103-bundle-runtime.md

```diff
@@ -140,11 +140,12 @@ The parameter value is evaluated thus:
 - If the CNAB runtime does not provide a value, but `default` is set, then the default value MUST be used.
 - If the parameter is marked `required` and `default` is set, then the requirement is satisfied by the runtime-provided default.
 - If no value is provided and `default` is unset, the runtime MUST set the value to an empty string (""), regardless of type.
-- Values are encoded as JSON strings.
+- Non-string values SHOULD be converted to [JSON text](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). For example, boolean `true` (or `True` or `TRUE`) SHOULD be expressed as `true` and the object `{foo: 23}` SHOULD be expressed as `{"foo":23}`. String values SHOULD NOT be converted to JSON text.
```

Comment:
Perhaps this is just temporary/coincidence, but the link here is currently hanging for me when using the `http` protocol, as written.  When I switch to `https`, all is well: https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf

Reply:
Fixed